core/src/components/button/test/clear/button.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('button: clear', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/button/test/clear`);
+
+    await page.setIonViewport();
+
+    expect(await page.screenshot({ fullPage: true })).toMatchSnapshot(`button-clear-${page.getSnapshotSettings()}.png`);
```

Comment:
`{ fullPage: true }` should not be needed when using `page.setIonViewport()`. This should be cleaned up in each button e2e test. 

Likely this was copy/paste from the original `button.e2e.ts` test. I think that was introduced before we realized it wasn't needed. Would be good to remove from that test as well.

Reply:
Good catch 👀